docs/src/getting-started.rst

```diff
@@ -16,6 +16,11 @@ Hello, |cdk|!
 
 This topic walks you through creating and deploying your first |cdk| app.
 
+.. note::
+
+    Unless otherwise indicated,
+    the code examples in this guide are in TypeScript.
```

Comment:
I am not sure that's the right place for this. The "getting started" topic actually has code in all supported languages. Perhaps this can go in "welcome". Also, if you are saying that, maybe we can provide some advice on how to "translate" the examples to one's language. For example, you can search the reference for the names of modules and classes in your language, etc.